.github/workflows/install_test.yml

```diff
@@ -0,0 +1,33 @@
+on:
+  pull_request:
+    types: [opened, synchronize]
```

Comment:
@jeff-hernandez I'm tempted to ask you to delete this, so that these tests only run on merges to `main`. But I guess these jobs will complete rather quickly, so it may be fine to just let them run.

@chukarsten @freddyaboulton tagging you RE recent discussions about our test suite and how many jobs we run on each PR.

I wish there were a way to say "run these tests on merges to `main`, and also if the update_checker.py code has been modified", but that's probably not worth the trouble.

A nice compromise could be to run one python version on PRs and the other two on merge to `main`.

Reply:
@dsherry there is a way to say "run these tests on merges to main, and also if the update_checker.py code has been modified"

```yaml
on:
  push:
    branches:
      - main
  pull_request:
    types: [opened, synchronize]
    paths:
      - 'evalml/update_checker.py'
```

Reply:
Woah. I would be a fan of us using that here! That way, if we modify this code in the future, the tests will still run in the PR :)

Reply:
I'll add that to the workflow